docs/documentation/native/native2.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: Native Automation 2.0 (Native2)
+---
+
+<Info>
+Native2 is available starting from Patrol version `3.6.0`.
+</Info>
+
+## What is `native2`?
+
+Native2 was created to address fundamental limitations in the original native automation approach. The original native API was primarily designed for Android, and attempts to make a single `Selector` work across both platforms proved problematic because:
+
+- **Platform-specific identifiers**: iOS and Android use different identifier systems (Android's `resourceName` vs iOS's `identifier`) and a single selector approach couldn't effectively handle the fundamental differences between iOS and Android element identification
+
+Native2 provides **platform-specific selectors** that work with both Android and iOS, giving you more accurate platform-specific selectors instead of one shared selector.
```

Comment:
This sentence is much better than the one above and I think it tells exactly what the previous sentence was trying to say